deploy/tasks/pre-build.yaml

```diff
@@ -103,7 +106,7 @@ spec:
           memory: 512Mi
       script: |
         $(params.BUILD_SCRIPT)
-        /opt/jboss/container/java/run/run-java.sh $(params.PREPROCESSOR_ARGS)
+        /opt/jboss/container/java/run/run-java.sh $(params.BUILD_TOOL)-prepare $(workspaces.source.path)/source --recipe-image=$(params.RECIPE_IMAGE) --request-processor-image=$(params.JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE) --disabled-plugins=$(params.BUILD_PLUGINS)
```

Comment:
This completely inlines the request processor arguments (which makes it easier for PNC to use)